.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, '3.10', 3.11, 3.12]
+        python-version: ['3.10', 3.11, 3.12, 3.13]
```

Comment:
```suggestion
        python-version: [3.10, 3.11, 3.12, 3.13]
```

Reply:
quote are mandatory, otherwise, it interpret it as '3.1' instead of '3.10'